From 5ca902a7d23e8d6ab62c469791a00c18c7893ae4 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Mon, 1 May 2017 18:22:04 +0200
Subject: [PATCH 30/39] extract_utils: Fix cleanup variables assigment

The current assignment is wrong, we cleanup the files when we tell
not to do so.

Change-Id: I73a9a17f5ebae6f2dbdc3d6d9aaa28cd75fb4f87
---
 build/tools/extract_utils.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 11819f3e..d241cf5a 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -92,7 +92,7 @@ function setup_vendor() {
         COMMON=0
     fi
 
-    if [ "$5" == "true" ] || [ "$5" == "1" ]; then
+    if [ "$5" == "false" ] || [ "$5" == "0" ]; then
         VENDOR_STATE=1
         VENDOR_RADIO_STATE=1
     else
-- 
2.15.0

